app/views/dashboard/_eeo_data.haml

```diff
@@ -4,7 +4,7 @@
     .note_item
       = select_tag :department_ids,
       options_from_collection_for_select(@departments, :id, :name),
-      class: 'dept_select', multiple: true
+      class: 'dept-select', multiple: true
```

Comment:
Sure. But if I change the class name to have an underscore in it, it stops working.